pyrate/core/geometry.py

```diff
@@ -128,19 +162,21 @@ def calc_pixel_geometry(ifg, rg, lon, lat, params):
     near_range = float(ifg.meta_data[ifc.PYRATE_NEAR_RANGE_METRES])
     rps = float(ifg.meta_data[ifc.PYRATE_RANGE_PIX_METRES])
     heading = float(ifg.meta_data[ifc.PYRATE_HEADING_DEGREES])
-    azimuth = float(ifg.meta_data[ifc.PYRATE_AZIMUTH_DEGREES])
+    # direction of look vector w.r.t. satellite heading. 
+    # Gamma convention: +ve = right; -ve = left.
+    right_or_left_look = float(ifg.meta_data[ifc.PYRATE_AZIMUTH_DEGREES])
 
     # Read height data from DEM
     dem_file = join(params[cf.OUT_DIR], 'dem.tif')
     DEM_data = DEM(dem_file)
     DEM_data.open(readonly=True)
     dem_height = DEM_data.height_data
 
-    # convert angles to radians
+    # convert to radians
     lon = np.radians(lon)
     lat = np.radians(lat)
     heading = np.radians(heading)
-    azimuth = np.radians(azimuth)
+    right_or_left_look = np.radians(right_or_left_look)
```

Comment:
maybe replace with 'look_direction'

Reply:
I prefer `look_dir`, as in above comment